training/algorithms.rst

```diff
@@ -105,15 +105,6 @@ deepar
    ../introduction_to_amazon_algorithms/deepar_electricity/DeepAR-Electricity
    ../introduction_to_applying_machine_learning/deepar_chicago_traffic_violations/deepar_chicago_traffic_violations
 
-
-linear_learner
---------------
-
-.. toctree::
-   :maxdepth: 1
-
-   ../introduction_to_applying_machine_learning/linear_time_series_forecast/linear_time_series_forecast
```

Comment:
Is there any other Linear Learner notebook we can highlight if this one was deleted?

Reply:
Unfortunately, I have searched and not found one. There are many linear learner examples but not one with time series application.